runtime/oti/shcdatatypes.h

```diff
@@ -131,70 +135,63 @@ typedef struct ClasspathWrapper {
 #define CPWDATA(cpw) ((U_8*)(cpw) + sizeof(ClasspathWrapper))
 #define CPWLEN(cpw) (sizeof(ClasspathWrapper) + J9SHR_READMEM((cpw)->ClasspathItemSize))
 
+typedef struct J9ShrOffSet {
```

Comment:
This should be named `J9ShrOffset` ('offset' is one word so the capital 'S' is inappropriate).
Likewise, parameters and variables should be named `offset` instead of `offSet`.

Reply:
Fixed